<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewRecord</class>
 <widget class="QDialog" name="ViewRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>                 Your Record</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_surgery">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableView" name="tableSurgery"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_addentry"/>
     <widget class="QWidget" name="page_diagnosis">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="tableDiagnosis"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_diary">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTableView" name="tableDiary"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_vaccination">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTableView" name="tableVaccination"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_prescription">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableView" name="tablePrescription"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_allergy">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableView" name="tableAllergy"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>View Diary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>View Surgical History </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>View Prescriptions </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>View Diagnosis History</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>View Vaccination</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>View Allergy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>